docs/source/advanced.rst

```diff
@@ -258,23 +258,217 @@ of :py:class:`scikeras.wrappers.BaseWrappers`. Currently, they are:
 
 All routed parameters will be available for hyperparameter tuning.
 
+For example:
+
+.. code:: python
+
+   clf = KerasClassifier(..., fit__batch_size=32, predict__batch_size=10000)
+
 Below are some example use cases.
 
-Example: multiple losses with routed parameters
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Compilation with routed parameters
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+SciKeras can compile optimizers, losses and metrics with routed parameters.
+This allows hyperparameter tuning with deeply nested parameters to optimizer, losses and metrics.
+You can use this feature both when you let SciKeras compile your Model and when you compile your own model
+within ``model_build_fn`` (by accepting the ``compile_kwargs`` parameter).
+
+Optimizer
++++++++++
 
 .. code:: python
 
-    from tensorflow.keras.losses import BinaryCrossentropy, CategoricalCrossentropy
+    from scikeras.wrappers import KerasClassifier
+    from tensorflow import keras
 
     clf = KerasClassifier(
         model=model_build_fn,
+        optimizer=keras.optimizers.SGD,
+        optimizer__learning_rate=0.05
+    )
+    clf = KerasClassifier(  # equivalent model
+        model=model_build_fn, optimizer=keras.optimizers.SGD(learning_rate=0.5)
+    )
+
+.. note::
+
+   Only routed parameters can be tuned; the object syntax can not be tuned.
+   That is, in the example above, ``optimizer__learning_rate`` can be tuned with
+   `sklearn.model_selection.RandomizedSearchCV`; `SGD(learning_rate=0.5)` can
```

Comment:
> That is, in the example above, it's possible to tune the learning rate with Scikit-Learn's `RandomizedSearchCV` by specifying `optimizer__learning_rate`. It's not possible to tune the learning rate with Scikit-Learn when `optimizer=SGD(learning_rate=0.5)` is specified.

There's also some double backticks missing (so text that should be monospaced will be italicized).

Reply:
Great, thanks for the better text and catching the backticks.